dns.c

```diff
@@ -380,6 +501,24 @@ parse_question(struct interface *iface, struct sockaddr *from, char *name, struc
 		break;
 
 	case TYPE_PTR:
+		host = strstr(name, ".in-addr.arpa");
+		if (host) {
```

Comment:
this should only match if ".in-addr.arpa" is at the end of the string, right ?

Reply:
thanks for reviewing this @blogic , code updated as per comment.